configs/config.go

```diff
@@ -21,41 +21,34 @@ type Config struct {
 	// This is a common option when the container is running in ramdisk
 	NoPivotRoot bool `json:"no_pivot_root,omitempty"`
 
+	// ParentDeathSignal specifies the signal that is sent to the container's process in the case
+	// that the parent process dies.
+	ParentDeathSignal int `json:"parent_death_signal,omitempty"`
+
 	// PivotDir allows a custom directory inside the container's root filesystem to be used as pivot, when NoPivotRoot is not set.
 	// When a custom PivotDir not set, a temporary dir inside the root filesystem will be used. The pivot dir needs to be writeable.
 	// This is required when using read only root filesystems. In these cases, a read/writeable path can be (bind) mounted somewhere inside the root filesystem to act as pivot.
 	PivotDir string `json:"pivot_dir,omitempty"`
 
-	// ReadonlyFs will remount the container's rootfs as readonly where only externally mounted
-	// bind mounts are writtable
-	ReadonlyFs bool `json:"readonly_fs,omitempty"`
+	// Path to a directory containing the container's root filesystem.
+	Rootfs string `json:"rootfs,omitempty"`
```

Comment:
Well we are changing alot of things in this branch, moving things from config to process, etc.  It would be nice to make the changes that we want now and then not worry about it in the future.

I was thinking at the end i'll check in a `configs/v1` pkg that has the current config and write a function to migrate to and from the current config to our new struct.